feature-libs/customer-ticketing/root/customer-ticketing-root.module.ts

```diff
@@ -0,0 +1,5 @@
+import { NgModule } from '@angular/core';
+import {} from '@spartacus/core';
+
+@NgModule({})
```

Comment:
Align with @Parthlakhani on the trigger cms components to lazy load this feature, so that it can be properly worked on when this gets merged